src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -14,6 +14,7 @@ namespace ov {
 namespace snippets {
 namespace lowered {
 
+#ifdef ENABLE_OPENVINO_DEBUG
```

Comment:
@chenhu-wang  @IvanNovoselov  
Major proposal from my side is to add new defininition `SNIPPETS_DEBUG_CAPS` which will be full analogue of `CPU_DEBUG_CAPS` and use it instead of `ENABLE_OPENVINO_DEBUG`.
On cmake level we already have `ENABLE_DEBUG_CAPS` option enablement will trigger both Snippet and Cpu debug capabilities underneath. That should be more convenient for developers.

Reply:
@dmitry-gorokhov , updated in [d83014e](https://github.com/openvinotoolkit/openvino/pull/21494/commits/d83014eaafcb464ee1c49f6e242691302c75c83b). Thanks Dmitry.